kafka-connect/kafka-connect-runtime/src/integration/java/org/apache/iceberg/connect/TestContext.java

```
@@ -51,6 +52,7 @@ public class TestContext {
   private TestContext() {
     ComposeContainer container =
         new ComposeContainer(new File("./docker/docker-compose.yml"))
+            .withStartupTimeout(Duration.ofMinutes(2))
```

How is this timeout related to test failures like `ExceptionInInitializerError`?

---

My hope is this will give more time for the containers to initialize before failing and retrying.

---

If containers don't initialize in time, the tests can continue to run and fail eventually?

---

It looked like the tests were failing to initialize the containers before running.

---

> It looked like the tests were failing to initialize the containers before running.

yeah that's also what I was seeing when I looked into this a bit

---

Can we fail early when the containers are not initialized? The current exceptions look really confusing.

---

I moved the context init to `BeforeAll` so the test will fail at initialization, which should be more clear.